src/snowflake/snowpark/session.py

```diff
@@ -708,28 +708,6 @@ def auto_clean_up_temp_table_enabled(self) -> bool:
         :meth:`DataFrame.cache_result` in the current session when the DataFrame is no longer referenced (i.e., gets garbage collected).
         The default value is ``False``.
 
-        Example::
```

Comment:
make sure we got a ticket to add this back once we are ready to rollout

Reply:
yes https://snowflakecomputing.atlassian.net/browse/SNOW-1738922